core/signal/signame_spec.rb

```diff
@@ -0,0 +1,25 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Signal.signame" do
+  it "takes a signal name with a well known signal number" do
+    Signal.signame(0).should == "EXIT"
+    Signal.signame(1).should == "HUP"
+    Signal.signame(2).should == "INT"
+  end
```

Comment:
I didn't want to depend on `Signal.list`.
But, It's not strong opinion.
I'll use `Signal.list`.